js/source/sidebar.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file
+ * A JavaScript file for the theme.
+ *
+ * In order for this JavaScript to be loaded on pages, see the instructions in
+ * the README.txt next to this file.
+ */
+
+// JavaScript should be made compatible with libraries other than jQuery by
+// wrapping it with an "anonymous closure". See:
+// - https://drupal.org/node/1446420
+// - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
+(function (Drupal, $, window) {
+
+  // To understand behaviors, see https://www.drupal.org/node/2269515
+  Drupal.behaviors.basic = {
```

Comment:
Good point - but I copied this js template file from stanford_basic, which in turn copied it from drupal core basic theme.  I agree with you, this isn't best practice, but at least I'm in good company!

I can fix it in this use case at least.